.github/workflows/tests.yml

```diff
@@ -61,3 +61,35 @@ jobs:
           -sdk ${{ matrix.sdk }} \
           -destination ${{ matrix.destination }} \
           test-without-building
+
+  swift-button-functional-test:
+    runs-on: macOS-latest
+    defaults:
+      run:
+        working-directory: Samples/Swift/DaysUntilBirthday
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Create xcconfig file
+      shell: bash
+      env:
+        EMAIL_SECRET : ${{ secrets.EMAIL_SECRET }}
```

Comment:
Remind me what type of Gaia account we're currently using with these secrets?

Reply:
It's an owned test account that I have.